media/css/devreg/reviewers.styl

```diff
@@ -1092,6 +1092,19 @@ table.activity .error {
     }
 }
 
+.review-actions-testedon {
+    .formset {
+        margin-top: 1.5em;
+    }
+    a.add-row {
+        background: url("../../img/icons/news-feed-sprite.png") no-repeat 0 -1444px;
```

Comment:
I reduced it down to the 3 (I decided on a different one for adding a new line) icons and deleted the original.  We probably have a *lot* of unused image files in media (different bug)